Aptoide/aptoide-client-v8
# [ASV-1523] Fix unity ads

**What does this PR do?**
This PR aims at fixing unity ads. MoPub team sent us a sample project in order for us check if there is any difference in the integration.
The difference that i noticed was that we were importing unity ads through aar (manual integration) instead of adding the dependency through gradle.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] build.gradle app

**How should this be manually tested?**
Open the app and check if you have ads. The only way for us to see if the fix really worked for unity ads is through the mopub dashboard. We will release it and check if the numbers improve.

**Also, since we increased version of flurry, please check if flurry events are working correctly by testing if a flurry event is being correctly sent.**

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1523](https://aptoide.atlassian.net/browse/ASV-1523)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
